uniffi_bindgen/src/interface/enum_.rs

```diff
@@ -113,6 +113,14 @@ impl Enum {
         // and its contained types could use a bit of a cleanup.
         ci.type_contains_object_references(&Type::Enum(self.name.clone()))
     }
+
+    pub fn type_contains_unsigned_types(&self, ci: &ComponentInterface) -> bool {
```

Comment:
Reverse of the previous naming nit 😅 - on these structs we've now got `contains_object_references` without a `type_` prefix and `type_contains_unsgined_types` with one. I lightly prefer to remove the `type_` prefix on the name here because the subject of the predicate is the method receiver itself, so it'll get called like `e.contains_unsigned_types` which I think reads succinctly.